tests/system/common.py

```diff
@@ -16,6 +16,7 @@
 from json.decoder import JSONDecodeError
 from shakedown import marathon
 from urllib.parse import urljoin
+from utils import get_cluster_slave_domains, get_used_regions_and_zones, get_app_domains
```

Comment:
I'm pretty sure flake8 does not like this 😉.

Reply:
I see no complain from `flake8` on this 😄